beacon-chain/sync/backfill/verify.go

```diff
@@ -27,7 +58,7 @@ type verifier struct {
 }
 
 // TODO: rewrite this to use ROBlock.
```

Comment:
is this TODO comment still needed?

Reply:
yeah it's referring to the input parameter.